modules/seqexec/server/src/main/scala/seqexec/server/gmos/GmosInstrumentActions.scala

```diff
@@ -48,6 +49,7 @@ class GmosInstrumentActions[F[_]: MonadError[?[_], Throwable]: Concurrent: Logge
     r match {
       case ObserveCommandResult.Success =>
         okTail(fileId, dataId, stopped = false, env)
+          .as(Result.Partial(NSFinalObs)) // For normally completed observations send a partial
```

Comment:
This will let us send messages after the obs completes. It needs to be matched with an `Ok` down below